site/content/zh-CN/docs/concepts/workload_priority_class.md

```diff
@@ -74,42 +71,37 @@ spec:
-- `WorkloadPriorityClass` is used for the workload's priority.
-- `PriorityClass` is used for the pod's priority.
-2. A job specifies only `WorkloadPriorityClass`
-- `WorkloadPriorityClass` is used for the workload's priority.
-- `WorkloadPriorityClass` is not used for pod's priority.
-3. A job specifies only `PriorityClass`
-- `PriorityClass` is used for the workload's priority and pod's priority.
-
-In certain job frameworks, there are CRDs that:
-- Define multiple pod specs, where each can have their own pod priority, or
-- Define the overall pod priority in a dedicated field.
-By default kueue will take the PriorityClassName of the first PodSet having one set,
-however the integration of the CRD with Kueue can implement
-[`JobWithPriorityClass interface`](https://github.com/kubernetes-sigs/kueue/blob/e162f8508b503d20feb9b31fd0b27d91e58f2c2f/pkg/controller/jobframework/interface.go#L81-L84)
-to change this behavior. You can read the code for each job integration
-to learn how the priority class is obtained.
-
-## Where workload's priority is used
-
-The priority of workloads is used for:
-- Sorting the workloads in the ClusterQueues.
-- Determining whether a workload can preempt others.
-
-## Workload's priority values are always mutable
-
-The `Workload`'s `Priority` field is always mutable.
-If a `Workload` has been pending for a while, you can consider updating its priority to execute it earlier,
-based on your own policies.
-Workload's `PriorityClassSource` and `PriorityClassName` fields are immutable.
-
-## What's next?
-
-- Learn how to [run jobs](/docs/tasks/run/jobs)
-- Learn how to [run jobs with workload priority](/docs/tasks/manage/run_job_with_workload_priority)
-- Read the [API reference](/docs/reference/kueue.v1beta1/#kueue-x-k8s-io-v1beta1-WorkloadPriorityClass) for `WorkloadPriorityClass`
+## Pod 优先级与 Workload 优先级的关系
+
+为给定作业创建 `Workload` 时，Kueue 会考虑以下场景：
+1. 作业同时指定了 `WorkloadPriorityClass` 和 `PriorityClass`
+- `WorkloadPriorityClass` 用于 workload 的优先级。
+- `PriorityClass` 用于 pod 的优先级。
+2. 作业仅指定了 `WorkloadPriorityClass`
+- `WorkloadPriorityClass` 用于 workload 的优先级。
+- `WorkloadPriorityClass` 不用于 pod 的优先级。
+3. 作业仅指定了 `PriorityClass`
+- `PriorityClass` 用于 workload 和 pod 的优先级。
+
+在某些作业框架中，CRD 可能：
+- 定义多个 pod 规范，每个规范可以有自己的 pod 优先级，或
+- 在专用字段中定义整体 pod 优先级。
+默认情况下，kueue 会采用第一个设置了 PriorityClassName 的 PodSet，但 CRD 与 Kueue 的集成可以实现 [`JobWithPriorityClass 接口`](https://github.com/kubernetes-sigs/kueue/blob/e162f8508b503d20feb9b31fd0b27d91e58f2c2f/pkg/controller/jobframework/interface.go#L81-L84) 来改变此行为。你可以阅读每个作业集成的代码，了解优先级类的获取方式。
+
+## Workload 优先级的用途
+
+Workload 的优先级用于：
+- 在 ClusterQueue 中对 workload 排序。
+- 决定某个 workload 是否可以抢占其他 workload。
+
+## Workload 优先级值始终可变
+
+`Workload` 的 `Priority` 字段始终是可变的。
+如果某个 `Workload` 已经挂起一段时间，你可以根据自己的策略考虑提升其优先级以提前执行。
+Workload 的 `PriorityClassSource` 和 `PriorityClassName` 字段是不可变的。
+
+## 下一步？
+
+- 了解如何[运行作业](/docs/tasks/run/jobs)
+- 了解如何[使用 workload 优先级运行作业](/docs/tasks/manage/run_job_with_workload_priority)
```

Comment:
```suggestion
- 了解如何[使用 Workload 优先级运行作业](/docs/tasks/manage/run_job_with_workload_priority)
```

Reply:
I have corrected the mentioned errors, including mistakes such as capitalization, line breaks, and indentation.